content/v2.0/actions/formats-and-mime-types.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Formats and MIME types
+order: 70
+---
+
+Hanami maps [over 50 of the most common MIME types][built-in-formats] to simple **format** names for you to use when configuring your actions.
+
+[built-in-formats]: https://github.com/hanami/controller/blob/dc5bb2a1db48b0ccf3faf52aac20eaef0fd135a3/lib/hanami/action/mime.rb#L15-L69
+
+Configuring one or more formats for your actions will:
+
+- Ensure the actions accept only appropriate requests based on their `Accept` or `Content-Type` headers
+- Set an appropriate `Content-Type` header on responses
+- For certain formats, enable automatic parsing of request bodies
+
+## Configuring a format for all actions
+
+To configure a format for all actions, use `config.actions.format` in your app class.
+
+```ruby
+# config/app.rb
+
+module Bookshelf
+  class App < Hanami::App
+    config.actions.format :json
+  end
+end
+```
+
+You can also configure actions to use multiple formats:
+
+```ruby
+config.actions.format :json, :html
+```
+
+## Configuring a format for particular actions
+
+You can also configure a format on any action class. `format` in an action class is analogous to `config.actions.format` in your app class, just as `config` in an action is analogous to `config.actions` in your app.
```

Comment:
```suggestion
You can also configure a format on any action class. `format` in an action class is analogous to `config.actions.format` in your app class, just as `config` in an action is analogous to `config.actions` in your app class.
```

Reply:
I did leave that off purposefully to avoid tiresome repetition :) Can add it back if you think it's important!